src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -172,6 +172,14 @@ export const ControlGroupStrings = {
       i18n.translate('controls.controlGroup.management.addControl', {
         defaultMessage: 'Add control',
       }),
+    getApplyButtonTitle: (applyResetButtonsEnabled: boolean) =>
+      applyResetButtonsEnabled
+        ? i18n.translate('controls.controlGroup.management.applyButtonTooltip.enabled', {
+            defaultMessage: 'Apply selections',
+          })
+        : i18n.translate('controls.controlGroup.management.applyButtonTooltip.disabled', {
+            defaultMessage: 'No new selections to apply',
+          }),
```

Comment:
Copy help

Reply:
@amyjtechwriter

Reply:
I think we'd better keep 'selections' @Heenawter for consistently, but I understand filters would also work really well.

Reply:
Amazing, thank you!! 🙇